R/use_roclets.R

```diff
@@ -0,0 +1,63 @@
+
+#' Use roclets to augment package compilaion.
+#'
+#' @param use_roxygen_roclets Logical (default: `TRUE`), indicates
+#' if default `roxygen2` roclets should be referenced in the DESCRIPTION file.
```

Comment:
Not sure I understand what this means.

Reply:
roxygen2 provides several `default` roclets, e.g. `roxygen2::rd_roclet` and `roxygen2::namespace_roclet`. These are responsible for parsing tags and writing docs & exports. If we specify custom roclets in `DESCRIPTION`, these default ones are not called.
[Here](https://github.com/extendr/rextendr/issues/59#issuecomment-796728016) I got an explanation of how to add these default roclets:
> 
> 
> @Ilia-Kosenkov you can get the default roclets with `roxygen2::roxy_meta_get("roclets")`, so something like `Roxygen: list(roclets = c(rextendr::my_roclet, roxygen2::roxy_meta_get("roclets")))` should run the rextendr roclet along with the default roclets.

So, this flag controls if `roxygen2::roxy_meta_get("roclets")` is written to the `DESCRIPTION` file. Not sure we will ever need this, but it is good to have configurable options.

Anyway, I'll write more comments & explanations.